client/components/select-dropdown/index.jsx

```diff
@@ -275,6 +278,7 @@ class SelectDropdown extends Component {
 	}
 
 	toggleDropdown() {
+		if ( this.props && this.props.disabled ) return;
```

Comment:
Let's add curly brackets here to be safe.

Reply:
@ehg Fixed ✔️